### PR TITLE
Adding Ansys Mechanical App

### PR DESCRIPTION
**Pull request by JonShelley**
I have added the Ansys Mechanical app. This will allow users to run the standard benchmarks with some minor changes.

### CODE REVIEW

File: apps/ansys_mechanical/readme.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+azhpc-scp -u hpcuser -r $azhpc_dir/apps hpcuser@headnode:.
+```
+
+## Connect to the headnode
+
+```
+azhpc-connect -u hpcuser headnode
+```
+
+## Install Ansys Mechanical
```
Comment (kanchanm) [inline apps/ansys_mechanical/readme.md]: should use azhpc-run to be consistent with the other app integration

File: apps/ansys_mechanical/readme.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+Finally, add "add_pkg" to the tags section for the compute nodes (~line 50 in the config.json file).Once these changes are made, then when you build the cluster (azhpc-build) it will get the neccessary scripts to install the prerequsites on the compute nodes
+
+Dependencies for binary version:
+
+* v19.*
+
+## Installation
+
+NOTE: Update the path to the fluent installer tar file in $azhpc_dir/apps/fluent/install_fluent.sh
```
Comment (kanchanm) [inline apps/ansys_mechanical/readme.md]: mechanical?

Comment (xpillons) [reply apps/ansys_mechanical/readme.md]: should you just refer to the right tutorial configuration instead of documenting the config update ?

Comment (JonShelley) [reply apps/ansys_mechanical/readme.md]: Done.

File: apps/ansys_mechanical/readme.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+### Install Prerequisite Software
+```
+azhpc-run -u hpcuser -n "headnode compute" ~/apps/ansys_mechanical/scripts/add_reqs.sh
+```
+
+### Install Ansys Mechanical
+```
+azhpc-run -u hpcuser -n headnode ~/apps/ansys_mechanical/install_mechanical.sh \<tar file name\> \<URL to tar file of the installer\>
```
Comment (kanchanm) [inline apps/ansys_mechanical/readme.md]: @edwardsp - how does azhpc-run handle this? can you pass args to the script here?

**Comment by xpillons** [comment]
@JonShelley please merge and delete that branch

**Pull request merged**